common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -249,6 +251,7 @@ def student_view(self, context):
             'ajax_url': self.system.ajax_url,
             'next_url': context.get('next_url'),
             'prev_url': context.get('prev_url'),
+            'masquerade_override': masquerading and view_html is not None,
```

Comment:
Can we come up with a better name here?  What information are we exactly passing to the mako template?  Isn't it more like 'show_hidden_timed_exam`?